tools/styles/body.ts

```diff
@@ -11,23 +11,7 @@ governing permissions and limitations under the License.
 */
 
 import baseStyles from './src/spectrum-base.css.js';
-import langBaseStyles from './src/spectrum-lang.css.js';
```

Comment:
None of the `*-overrides` files were necessary for typography so this reduces that complexity. Note the files are not removed because they're being shipped in the exports object and thus could be used by customers downstream without us knowing. The files will now be empty but present.